### PR TITLE
add click show detailed data

### PR DESCRIPTION
**Pull request by keien411**
1. add click show detailed data
2. view display srea boundary value drawing. always show the now price

### CODE REVIEW

File: lib/renderer/base_chart_painter.dart [hunk 1]
```diff
@@ -50,6 +51,7 @@ abstract class BaseChartPainter extends CustomPainter {
     required this.scaleX,
     required this.scrollX,
     required this.isLongPress,
+    required this.isOnTap,
```
Comment (mafanwei) [inline lib/renderer/base_chart_painter.dart:54]: Maybe some people don’t need this feature.
Can it be null?

**Comment by mafanwei** [comment]
Thank you for your contribution, I have review it.

File: lib/renderer/base_chart_painter.dart (hunk 1 above)
Comment (keien411) [reply lib/renderer/base_chart_painter.dart:54]: sure, I can modify

Comment (keien411) [reply lib/renderer/base_chart_painter.dart]: I Add a switch to click to show info data, If someone doesn’t need it, can turn it off.
Default is off

**Pull request merged**